src/character.cpp

```diff
@@ -12398,13 +12399,14 @@ stat_mod Character::get_pain_penalty() const
 {
     stat_mod ret;
     int pain = get_perceived_pain();
-    if( pain <= 0 ) {
+    // if less than 10 pain, do not apply any penalties
+    if( pain <= 10 ) {
         return ret;
     }
 
     // Int and per are penalized more, 100 pain to drop stat to zero vs 140-ish for str and dex
     float penalty_mod = pain * 0.01f;
-    float lesser_penalty_mod = pain * 0.007f;
+    float lesser_penalty_mod = pain * 0.005f;
 
 
     ret.strength = enchantment_cache->modify_value( enchant_vals::mod::PAIN_PENALTY_MOD_STR,
```

Comment:
To keep from having too many variables for penalties, maybe strength could use the lesser_penalty_mod, and dexterity could use `avg(penalty_mod, lesser_penalty_mod)`. This is particularly a good way to do it if we define these as global json settings.

Reply:
I'm not sure now, should it be two variables then? Maybe just make three separate one? Or even four, because why this half measurements? We won't be able to make it a single variable anyway